google-cloud-video-live_stream/samples/acceptance/helper.rb

```diff
@@ -19,33 +19,146 @@
 require "google/cloud/video/live_stream"
 
 require_relative "../../../.toys/.lib/sample_loader"
+require_relative "channel_definition"
+require_relative "input_definition"
+
+DELETION_THRESHOLD_TIME_HOURS_IN_KILOSECONDS = 10_800_000
 
 class LiveStreamSnippetSpec < Minitest::Spec
   let(:client) { Google::Cloud::Video::LiveStream.livestream_service }
 
   let(:project_id) { ENV["GOOGLE_CLOUD_PROJECT"] || raise("missing GOOGLE_CLOUD_PROJECT") }
-
   let(:location_id) { "us-central1" }
+  let(:location_path) { client.location_path project: project_id, location: location_id }
+
   let(:input_id) { "my-input-test-#{(Time.now.to_f * 1000).to_i}" }
   let(:input_name) { "projects/#{project_id}/locations/#{location_id}/inputs/#{input_id}" }
-  let(:input_location_path) { client.location_path project: project_id, location: location_id }
+  let(:input_path) { client.input_path project: project_id, location: location, input: input_id }
+
+  let(:update_input_id) { "my-update-input-test-#{(Time.now.to_f * 1000).to_i}" }
+  let(:update_input_name) { "projects/#{project_id}/locations/#{location_id}/inputs/#{update_input_id}" }
+  let(:update_input_path) { client.input_path project: project_id, location: location, input: update_input_id }
+
+  let(:channel_id) { "my-channel-#{(Time.now.to_f * 1000).to_i}" }
+  let(:channel_name) { "projects/#{project_id}/locations/#{location_id}/channels/#{channel_id}" }
+  let(:output_uri) { "gs://my-bucket/my-output-folder/" }
+
+  attr_writer :channel_created_started
+  attr_writer :channel_created_stopped
+  attr_writer :input_created
+  attr_writer :update_input_created
+
+  before do
+    @channel_created_started = false
+    @channel_created_stopped = false
+    @input_created = false
+    @update_input_created = false
+    # Remove old channels and inputs in the test project if they exist
+    response = client.list_channels parent: location_path
+    response.each do |channel|
+      tmp = channel.name.to_s.split "-"
+      create_time = tmp.last.to_i
+      now = (Time.now.to_f * 1000).to_i # Kiloseconds, preserves float value for precision
```

Comment:
Again, this is milliseconds, not kiloseconds.

Reply:
Done